### DISCUSSION TITLE
AlphaDIA crash

### DISCUSSION THREAD

**Discussion by JM-Bader** [MannLabs/alphadia#124] [Show and tell]
It appears a session of AlphaDIA 1.5.5 has crashed - the program was closed. 
Ther is an error at the end of the log. Find it enclosed
[log.txt](https://github.com/MannLabs/alphadia/files/14978891/log.txt)

**Comment by GeorgWa** [comment]
Hi,

Thank you!

Can you check if it:

1. Reproducibly crashes with this library/ file combination
2. Reproducibly crashes after processing a large number of files with this
library.

Best,
Georg

**Comment by mschwoer** [comment]
should be fixed in the latest version
